### PR TITLE
Adxcg: support for mType and bidResponse currency

### PR DESCRIPTION
**Pull request by adxcgcom**

### CODE REVIEW

**Review by gargcreation1992 — DISMISSED** [review]
lgtm!

File: adapters/adxcg/adxcg.go [hunk 1]
```diff
@@ -105,21 +105,17 @@ func (adapter *adapter) MakeBids(
...
+		return openrtb_ext.BidTypeBanner, nil
+	} else if mType == openrtb2.MarkupVideo {
+		return openrtb_ext.BidTypeVideo, nil
+	} else if mType == openrtb2.MarkupNative {
+		return openrtb_ext.BidTypeNative, nil
+	} else if mType == openrtb2.MarkupAudio {
+		return openrtb_ext.BidTypeAudio, nil
+	} else {
+		return "", &errortypes.BadServerResponse{
+			Message: "Unsupported return type"}
```
Comment (onkarvhanumante) [inline adapters/adxcg/adxcg.go:119]: could make use of switch case here

```
switch mType {
    case openrtb2.MarkupBanner:
       return openrtb_ext.BidTypeBanner, nil
	case openrtb2.MarkupVideo:
		return openrtb_ext.BidTypeVideo, nil
	case openrtb2.MarkupNative:
		return openrtb_ext.BidTypeNative, nil
	case openrtb2.MarkupAudio:
		return openrtb_ext.BidTypeAudio, nil
	default:
		return "", &errortypes.BadServerResponse{Message: "Unsupported return type"}
}
```

Comment (adxcgcom) [reply adapters/adxcg/adxcg.go]: done

File: adapters/adxcg/adxcg.go [hunk 2]
```diff
@@ -105,21 +105,17 @@ func (adapter *adapter) MakeBids(
...
-				return openrtb_ext.BidTypeVideo, nil
-			}
-
-		}
-	}
-
-	return "", &errortypes.BadInput{
-		Message: fmt.Sprintf("Failed to find native/banner/video impression \"%s\" ", impID),
+func getReturnTypeFromMtypeForImp(mType openrtb2.MarkupType) (openrtb_ext.BidType, error) {
+	switch mType {
```
[the remark this answers is not in the document]
Comment (onkarvhanumante) [reply adapters/adxcg/adxcg.go:109]: yes this can be ignored

**Review by onkarvhanumante — APPROVED** [review]

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**